### PR TITLE
Prepare release notes for Maven 3.9.0

### PR DESCRIPTION
**Pull request by cstamas**
Some links (announce) are still just copy-pasta...

### CODE REVIEW

File: content/markdown/docs/3.9.0/release-notes.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+- the web site: [https://maven.apache.org/][2]
+- the maven-user mailing list: [https://maven.apache.org/mailing-lists.html](/mailing-lists.html)
+- the reference documentation: [https://maven.apache.org/ref/3.9.0/](/ref/3.9.0/)
+
+## Overview About the Changes
+
+* Minimum Java version to use with Maven 3.9.0 is raised to Java 8.
+* With Java 8, upgrade of several key dependencies became possible as well.
+* Regression fixes from Maven 3.8.x line.
```
Comment (michael-o) [inline content/markdown/docs/3.9.0/release-notes.md:43]: Do we really have regression fixes? I mean 3.9.0 should contain actually new stuff, no?

File: content/markdown/docs/3.9.0/release-notes.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+* General fixes and improvements.
+
```
Comment (michael-o) [inline content/markdown/docs/3.9.0/release-notes.md:45]: I'd also mention that it contains some backports from 4.

File: content/markdown/docs/history.md.vm [hunk 3]
```diff
@@ -89,7 +89,8 @@ Date format is: YYYY-MM-DD
 <!--  release( date version announce latest-minor jdk rowspan ) -->
 <!-- The following two lines needs to be fixed in x days if the archive picks up the announcement mail. -->
 
-#release( "2022-12-24" "3.8.7" "announce/202212.mbox/%3C27650d65-28f1-f3d8-472d-7d1939e1c83f@apache.org%3E" "true" "Java 7" "20" )
+#release( "2023-01-31" "3.9.0" "announce/202212.mbox/%3C27650d65-28f1-f3d8-472d-7d1939e1c83f@apache.org%3E" "true" "Java 8" "20" )
+#release( "2022-12-24" "3.8.7" "announce/202212.mbox/%3C27650d65-28f1-f3d8-472d-7d1939e1c83f@apache.org%3E" "true" "Java 7" "" )
```
Comment (hboutemy) [inline content/markdown/docs/history.md.vm:93]: "Java 7" "20"

File: content/markdown/docs/history.md.vm [hunk 4]
```diff
@@ -89,7 +89,8 @@ Date format is: YYYY-MM-DD
 <!--  release( date version announce latest-minor jdk rowspan ) -->
 <!-- The following two lines needs to be fixed in x days if the archive picks up the announcement mail. -->
 
-#release( "2022-12-24" "3.8.7" "announce/202212.mbox/%3C27650d65-28f1-f3d8-472d-7d1939e1c83f@apache.org%3E" "true" "Java 7" "20" )
+#release( "2023-01-31" "3.9.0" "announce/202212.mbox/%3C27650d65-28f1-f3d8-472d-7d1939e1c83f@apache.org%3E" "true" "Java 8" "20" )
```
Comment (hboutemy) [inline content/markdown/docs/history.md.vm:92]: "Java 8" "1"

**Review by hboutemy — CHANGES_REQUESTED** [review]

File: content/markdown/docs/3.9.0/release-notes.md (hunk 1 above)
Comment (cstamas) [reply content/markdown/docs/3.9.0/release-notes.md:43]: Look at release notes, I do see issues named as "regression" https://issues.apache.org/jira/secure/ReleaseNote.jspa?projectId=12316922&version=12350913

Comment (michael-o) [reply content/markdown/docs/3.9.0/release-notes.md:44]: Those regressions have also been applied to 3.8.7 as well. So there are no regressions between 3.8.x and 3.9.0. Both have the same fixes set.

Comment (cstamas) [reply content/markdown/docs/3.9.0/release-notes.md:44]: Well, https://issues.apache.org/jira/browse/MNG-5222 ? Did not go thru, but am fine to remove this line...

Comment (michael-o) [reply content/markdown/docs/3.9.0/release-notes.md:44]: Yes, but those aren't regressions, imho. Those are issues 3.8.x hasn't broken, but isn't able to fix due to Resolver and Java 7.

File: content/markdown/docs/3.9.0/release-notes.md [hunk 5]
```diff
@@ -41,7 +41,7 @@ If you have any questions, please consult:
 * Minimum Java version to use with Maven 3.9.0 is raised to Java 8.
 * With Java 8, upgrade of several key dependencies became possible as well.
 * Several backports from Maven 4.x line.
-* Regression fixes from Maven 3.8.x line.
+* Long outstanding issue fixes from Maven 3.8.x line.
```
Comment (michael-o) [inline content/markdown/docs/3.9.0/release-notes.md:44]: Very nice!

File: content/markdown/docs/history.md.vm [hunk 6]
```diff
@@ -89,6 +89,7 @@ Date format is: YYYY-MM-DD
 <!--  release( date version announce latest-minor jdk rowspan ) -->
 <!-- The following two lines needs to be fixed in x days if the archive picks up the announcement mail. -->
 
+#release( "2023-01-31" "3.9.0" "announce/202212.mbox/%3C27650d65-28f1-f3d8-472d-7d1939e1c83f@apache.org%3E" "true" "Java 8" "1" )
```
Comment (michael-o) [inline content/markdown/docs/history.md.vm:92]: Don't forget to update this link!

**Review by michael-o — APPROVED** [review]

**Review by hboutemy — APPROVED** [review]
LGTM, thanks for the hard work: releasing Maven core is not easy...

**Pull request merged**